### PR TITLE
Format code in FSharp.Compiler.Server.Shared and FSharp.DependencyManager.Nuget

### PR DESCRIPTION
**Pull request by dsyme**
Like the title says

### CODE REVIEW

File: src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs [hunk 1]
```diff
@@ -12,93 +12,155 @@ open FSharp.DependencyManager.Nuget.ProjectFile
...
 
     let formatPackageReference p =
-        let { Include=inc; Version=ver; RestoreSources=src; Script=script } = p
+        let {
+                Include = inc
+                Version = ver
+                RestoreSources = src
+                Script = script
+            } =
+            p
```
Comment (dsyme) [inline src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs:58]: These expansions of pattern matching on records are a really good reason not to use pattern matching on records! (I generally don't like the use of the feature).

File: src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs [hunk 2]
```diff
@@ -12,93 +12,155 @@ open FSharp.DependencyManager.Nuget.ProjectFile
...
+
         seq {
-            match not (String.IsNullOrEmpty(inc)), not (String.IsNullOrEmpty(ver)), not (String.IsNullOrEmpty(script)) with
-            | true, true, false  -> yield sprintf @"  <ItemGroup><PackageReference Include='%s' Version='%s' /></ItemGroup>" inc ver
-            | true, true, true   -> yield sprintf @"  <ItemGroup><PackageReference Include='%s' Version='%s' Script='%s' /></ItemGroup>" inc ver script
+            match not (String.IsNullOrEmpty(inc)), not (String.IsNullOrEmpty(ver)), not (String.IsNullOrEmpty(script))
+                with
```
Comment (dsyme) [inline src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs:62]: @nojaf seems strange to have the `with` on the new line

Comment (nojaf) [reply src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs:62]: I think this is stretching the limit and the match expression ends over the `max_line_length`.
The match expression is almost multiline, [sample](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKBbBBXANosAGmALwA6ATqQHYQSby74CaJF1tHYAzogI5jA2NTnQCGAF2gALMFVjiwACgDK48gEsqAcwB0ASS4A5XDgDy5AKLoADuICeizdACUzgDSz5S1Ru36jJuZWtg4Abojkrh5yCipqmroGxjhmljb2ilzQGrauYADu6uJSQiIQAD5gAPpgALQAfErOpWUAvkJC9IyErJTCdNh4YCxkfWXcfAItIugS0p6xPgn%2ByanBGU4AMrDaEVEL3vF%2BSYFpIYrh5Nu7ke4Hcb6JASlB6Q5ZOeLXWnvOBUUlMZlSo1BpNaacVqQNzgKA8cTiBJcVBgADasIgoBEkAAJPZrMg0JBNOJoRjaLjQqIcFhCWjyRwAAww8aQPRUJBUcTKdQAL2QLLKABYGWAALrk1qCzGi3H4unErlk7E4qk0uno8YARmlnEgAFlRAAPTaaRCbRDaYrKspagBMjNFEo4UvJWL1eLsBJRkEt8FMADNTVREKo7HgbRxKdTaSjNWU7bqoxYOYHgwLRRBIDgA5AnZLde6o57vUSAEawWA4SMU1UxjWZsAAZiTFPZPHI4gAYppqYZEPkcGaaxw1LT8y7C7KSwqK1WR1A6%2Bq442ha2s8prKJoIgAEKIAOwciIAAKonIonQiHEEQXEDHiAntFdHCLtflPrn1db0eXaHjIgAKzrlAm7bnuB5HuasD5BE0CiDw7KhLA8GIjsd5gA%2BT4QC%2BtBvlmM6fpW37Tmqsb-o2ABsIGQGBO77oex4AKrWAS5DwYhVDIah6joSBAbUjw2FgLhMoqh%2B5bEQuv7kfS4wAOw0XREGMYgADCOAIVwak7FwY7QOIR4YQJOBCSIzrPlO4lerOUk-kuskAZwAAcSlbvRkHHvqiDoGWt78YJj7mQWbrThJUBftJDkNuMACcbngQxUE6TgfGNiZZmcBZOFWR64WQJF9lkTFtrMo2tHuYgACCAY3uQOnoLMGFYcFk6hdZpYRXZpH1iu2o6uVylJceyg%2BeocCpVQxmBcJokCGFNlEfORW9RR2qJoNlU1XVo3oONVZpeMLVZSFr4LZ1BXdSqxV9baLabeBVXkLAWAcgAIogQ57XVzXkOOrWWe1eWLUSJJRTda22mu5WGkaegBgAKlIloWKZoZSEe4gAOrqPA1ogUKjoAzlQPFvlYMrX%2Bcm2sBMPGuyAbqEaphsRIR4WEa1jHlwXC8VNIGAUTJ1tWdHUKhTPVU05HBatRdNGgASogcDkPAON41IGGE7NuVkyDUAS9dq3UyIWqKfLSsq-Axi%2BREgZ6De6DIiBWo66T7765A6C4IiQ4hl2R6zAiEQI4tlOOY2WqueVltHvA%2Bo%2B%2BofuIAH5BBz9NHSOe251VUhQa3mxMibrHsXYbHoQybnBavF8tPRedjmKaenq-jq5Cxw2XF%2B7BHk0q4clSIDo0bD9eiI3VzqHpNt%2BeQ9uO87kdu6LwMXd7OC%2B2aqfpyHYeSxH4x2gNrJjxPzfiAnG9J1vgcSBn5VZxeBkRHnuPWsveHneL-f74PnB2htVksMABq9ZdyaHgAkVumsQLOQ7rQLuc18KLj7qSAet0h73SAcaLsr0DJ83ARyKBb8YHtw-mJVe380G-wwf-aG2CjRvXkAAcWvBzLmiAeZ82gRhQW5D5piyWiRI2UtGx2lpkAxOyddypWgAAa13E-OR14uCmCoMoS86kqxYHQPzdKM0i5IK-kI8Gxtpa0DtHLVk-ZBxmn3OIWClpQ4Eg%2BozKgRQ%2BZVQ5N5W25BF7jAykFYWgMV56wuoVGhkMh7mxPkOLQVBcFUHwTsHk8SJBYGPAjWA7JOTiAkHzaaaN%2BHILlJ7CJIiD4JmjrEuqVB8nhG%2BD42erjNAeN0oUzKndTqf0EZJZakSq4cDtLXSRV90ybEvGWeAogNKwF5toGxycOlBK6SLHplCTHoKiZwJsZVWTsIiOoK8XJqQAGlECIGsDky04h2SKNEEk0hASDHBJJqE0utl%2BkVL-hwJsx8yiQBkQ8uR6YuBPRehyfselEDxykWadh3NebtJAsdVZIT1lhM%2BcIiuZjGxNkAQC3c55hqIDelPbIRzew3ngExdxOwPrQE0heNCejnlFMMSXXuZSro4tEeMJsWCAUHI0McvJOAfAvT0n9awYYIwBXZa87u7yuVlx-t82hvz6EAvOZc2GM87YBiBVQEFZp-GlXgRARBnKUHcq%2BbyypIgmwSIBUsAy%2BoEAZjZZ0hBkpYRikFMSLgXZeYokCSAVoQA).
Related conversation: https://github.com/fsharp/fslang-design/issues/646

**Review by vzarytovskii — APPROVED** [review]

File: src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs (hunk 2 above)
Comment (dsyme) [reply src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.fs:62]: Fair enough, yes

File: src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs [hunk 3]
```diff
@@ -6,109 +6,119 @@ open System.IO
...
 
     let getResolutionsFromFile resolutionsFile =
 
         let lines =
             try
-                File.ReadAllText(resolutionsFile).Split([| '\r'; '\n'|], StringSplitOptions.None)
-                |> Array.filter(fun line -> not(String.IsNullOrEmpty(line)))
+                File
+                    .ReadAllText(resolutionsFile)
+                    .Split([| '\r'; '\n' |], StringSplitOptions.None)
```
Comment (vzarytovskii) [inline src/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs:89]: `File` on its own line looks weird.

**Pull request merged**